laravel/app/Http/Controllers/User/IndexController.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Http\Controllers\User;
+
+use App\Http\Controllers\Controller;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+use App\Models\Course;
+
+class IndexController extends Controller
+{
+    public function index()
+    {
+        $user = Auth::user();
+        $listCourse = Course::paginate(6);
```

Comment:
config number. vd nhuw
```suggestion
        $listCourse = Course::paginate(config('course.pagination'));
```